File: backend/src/main/java/pl/lahcimdev/comaco/dto/customer/CustomerDto.java
package pl.lahcimdev.comaco.dto.customer;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import pl.lahcimdev.comaco.customer.domain.CompanyContact;
import pl.lahcimdev.comaco.user.domain.Role;

import java.time.LocalDateTime;
import java.util.List;

@Getter
@Setter
@NoArgsConstructor

public class CustomerDto {

    private Long id;
    private String username;
    private List<Role> roles;
    private LocalDateTime createdDate;
    private String createdBy;
    private String companyName;
    private String companyStreet;
    private String companyPostalCode;
    private String companyCity;
    private String companyNip;
    private String companyRegon;
    private List<CompanyContact> contacts;
    private String photo;

}

File: frontend/src/api/models/customer-dto.ts
/* tslint:disable */
import { CompanyContact } from './company-contact';
import { Role } from './role';
export interface CustomerDto {
  companyCity?: string;
  companyName?: string;
  companyNip?: string;
  companyPostalCode?: string;
  companyRegon?: string;
  companyStreet?: string;
  contacts?: Array<CompanyContact>;
  createdBy?: string;
  createdDate?: string;
  id?: number;
  photo?: string;
  roles?: Array<Role>;
  username?: string;
}
